netci.groovy

```diff
@@ -26,6 +27,7 @@ def osGroupMap = ['Ubuntu14.04':'Linux',
 def targetNugetRuntimeMap = ['OSX' : 'osx.10.10-x64',
                              'Ubuntu14.04' : 'ubuntu.14.04-x64',
                              'Ubuntu16.04' : 'ubuntu.14.04-x64',
+                             'Fedora23' : 'ubuntu.14.04-x64',
```

Comment:
We don't publish fedora packages (yet) so restore would fail.  I'll address this later this week.